xml/api/resources-2.29.rng

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You'll want to clone a crm_resource-2.29.rng, crm_simulate-2.29.rng, and nodes-2.29.rng as well, since they all explicitly include the resources schema. Eventually I'd to automate that but for now it's manual.